ceph/rados/core_workflows.py

```diff
@@ -986,15 +986,6 @@ def verify_reweight(self, affected_osds: list, osd_info: list) -> bool:
 
         return True
 
-    def list_pools_on_cluster(self) -> list:
-        """
-        Method fetches the list of pools from the cluster via ceph df command
-
-        Returns:
-            list of pools on the cluster
-        """
-        return self.run_ceph_command(cmd="ceph osd pool ls", client_exec=True)
```

Comment:
Apologies for providing the incorrect command last time, I completely forgot about the extra line which gets printed here.
Also `run_ceph_command` would try to return a dictionary instead of a list so we shouldn't use it anyway.

Correct code - `return self.node.shell(["ceph osd pool ls -f json"])[0].strip("\n")`